lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -1451,6 +1451,8 @@ namespace Js
 
     RecyclableObject* JavascriptProxy::GetPrototypeSpecial()
     {
+        PROBE_STACK(GetScriptContext(), Js::Constants::MinStackDefault);
```

Comment:
Have a question related to this. JavascriptProxy::DeleteProperty does have PROBE_STACK but why SetProperty doesn't? Similarly IsExtensible which seems to have same pattern of CallFunction?